docs/usage/configuration-options.md

```diff
@@ -421,6 +421,31 @@ If enabled, all issues created by Renovate are set as confidential, even in a pu
 !!! note
     This option is applicable to GitLab only.
 
+## configMigration
+
+If enabled, Renovate will raise a pull request if config file migration is needed.
+
+We're adding new features to Renovate bot often.
+Most times you can keep using your Renovate config and benefit from the new features right away.
+But sometimes you need to change your Renovate configuration.
+To help you with this, Renovate will create config migration pull requests.
+
+Example:
+
+When we created the [`baseBranches`](https://docs.renovatebot.com/configuration-options/#basebranches) feature, the Renovate configuration migration pull request would look like this:
```

Comment:
```suggestion
After we changed the [`baseBranches`](https://docs.renovatebot.com/configuration-options/#basebranches) feature, the Renovate configuration migration pull request would make this change:
```

Now that we're showing a `diff` we should explain that this is the change the PR makes, not the content of the PR itself. 😉 

Also I think that we were not "creating a feature" but "changing the allowed input for an existing feature". So I think the new text is better. But check me on this. 😉